src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -395,6 +428,24 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     Return Nothing
                 End Get
             End Property
+
+            Public ReadOnly Property IsInvalid As Boolean Implements IOperation.IsInvalid
+                Get
+                    Return Me.Parameter Is Nothing OrElse Me.Value.IsInvalid
+                End Get
+            End Property
+
+            Public ReadOnly Property Kind As OperationKind Implements IOperation.Kind
+                Get
+                    Return OperationKind.Argument
+                End Get
+            End Property
+
+            Public ReadOnly Property Syntax As SyntaxNode Implements IOperation.Syntax
+                Get
+                    Return Me.Value.Syntax
+                End Get
+            End Property
         End Class
 
         Private Class ByRefArgument
```

Comment:
Would it be possible to extract common code into a base class for Argument and ByRefArgument?